packages/server/src/service/letter/index.ts

```diff
@@ -1,47 +1,21 @@
+import mailgun from 'mailgun-js'
 import marked from 'marked'
 import nunjucks from 'nunjucks'
 
 import { processEnvOrThrow, StateInfo, ContactMethod, ImplementedState } from '../../common'
 import { fillNewHampshire } from '../pdfForm'
 import { fillNorthCarolina } from '../pdfForm'
-import { makeImageAttachment } from '../mg'
+
+export const mg = mailgun({
+  domain: processEnvOrThrow('MG_DOMAIN'),
+  apiKey: processEnvOrThrow('MG_API_KEY'),
+})
 
 nunjucks.configure(__dirname + '/views', {
   autoescape: true,
   noCache: !!process.env.NUNJUNKS_DISABLE_CACHE
 })
 
-interface Options {
-  signature?: string
-  idPhoto?: string
-  form?: Buffer
-}
-
-export class Letter {
```

Comment:
Had to move Letter down on the file because eslint kept complaining about using functions before their declaration (which is a valid rule, but I don't think it should apply to class methods).